hexrd/ui/indexing/fit_grains_results_dialog.py

```diff
@@ -584,6 +598,60 @@ def on_grains_table_modified(self):
         self.add_extra_data_columns()
         self.update_plot()
 
+    def _save_workflow_files(self, selected_directory):
+
+        def full_path(file_name):
+            # Convenience function to generate full path via pathlib
+            return str(Path(selected_directory) / file_name)
+
+        HexrdConfig().working_dir = selected_directory
+
+        HexrdConfig().save_indexing_config(full_path('workflow.yml'))
+        HexrdConfig().save_materials(full_path('materials.h5'))
+        HexrdConfig().save_instrument_config(full_path('instrument.yml'))
+
+        ims_dict = HexrdConfig().unagg_images
+        for det in HexrdConfig().detector_names:
+            kwargs = {
+                'ims': ims_dict.get(det),
+                'name': det,
+                'write_file': full_path(f'{det}.h5'),
+                'selected_format': 'hdf5',
+                'path': 'imageseries',
+            }
+            HexrdConfig().save_imageseries(**kwargs)
```

Comment:
@joelvbernier The workflow config file, materials file, instrument file, and image series files all get saved here.

Reply:
Let me take a quick look right now!